.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules/
 npm-debug.log
-coverage/
```

Comment:
It was used by [istanbul](https://github.com/gotwarlost/istanbul), which is not used any longer